### PR TITLE
Remove user from repo if status can't be set

### PR DESCRIPTION
**Pull request by gylaz**

### CODE REVIEW

**Comment by gylaz** [comment]
[Investigate hound not setting status on PRs](https://trello.com/c/BM5wa1Px/848-investigate-hound-not-setting-status-on-prs)

File: spec/services/build_runner_spec.rb [hunk 1]
```diff
@@ -229,20 +229,22 @@ def force_fail_build_creation
     end
 
     context "when status cannot be updated" do
-      it "does not error" do
+      it "removes the user and re-raises the exception" do
```
[the remark this answers is not in the document]
Comment (salbertson) [reply spec/services/build_runner_spec.rb]: What does Hound want here?

**Comment by salbertson** [comment]
👍

File: spec/services/build_runner_spec.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (gylaz) [reply spec/services/build_runner_spec.rb]: This is Flog, it thinks that this block (method) is too large/complex.

**Comment by salbertson** [comment]
@gylaz what's the latest? Should this be closed for now?

**Comment by gylaz** [comment]
Yes, I'll close this.

To capture some info here: we can't safely do this, because on public repos `houndci-bot` isn't added to the repo, thus we might not have anyone's token that can set the status (however, can still review the public repo and leave comments). Because of this, I think we should leave it as-is until we move to GH App.

**Pull request closed without merging**